_episodes/l3_03_test_fixture.md

```diff
@@ -0,0 +1,505 @@
+---
+title: "Introduction to Test Fixtures using GoogleTest"
+teaching: TBC
+exercises: TBC
+questions:
+- "What is a test fixture?"
+- "How to create your own test fixture using GoogleTest?"
+- "What does Setup and Teardown mean while using test fixtures?"
+objectives:
+- "Understand the basics of test fixture."
+- "Understand how to create test fixture using GoogleTest."
+- "Understand the advantages of test fixtures."
+- "Understand the need of setup and teardown functions."
+- "Understand how to write your own setup and teardown function."
+- "Understand how to run a subset of tests."
+keypoints:
```

Comment:
This should be fleshed out

Reply:
If I understood clearly, you want more details to be added here (checked the meaning of phrase *flesh out* on Google). I do not think that we need to add more details here. This is the objective section and brief one liner should be OK. The explanation should be there in the text.

Reply:
I was referring to the "keypoints" section. I think this should provide a list of the main take-aways from the episode, rather than just re-stating the objectives.

Reply:
For the objectives, strinctly speaking, we should be using [Bloom's taxonomy](https://lib.guides.umd.edu/c.php?g=598357&p=4142337) in all these sections - in all of our courses. I think that the purpose of this course is largely to teach people to "apply", and not just "understand" which will fall a bit short of the expectations. @lkr-ind , maybe you can review some of the objectives using verbs from the "apply" section. 

About the keypoints, I agree with @dc2917 that they should be more take away messages, eg. "Setup and tear down functions let you configure the tests so they are reproducible and independent form each other", or something along those lines.

Reply:
@dc2917 @dalonsoa Thanks @dalonsoa for Bloom's taxonomy. I have made changes to keypoints and the objective section in faad5d5089b25497b670cab7cf40aa4c933645b5